tests/e2e/goto_definition.rs

```diff
@@ -68,17 +74,26 @@ fn test_goto_members(
             work_done_progress_params: Default::default(),
             partial_result_params: Default::default(),
         };
-        let goto_definition_response =
+        let response =
             ls.send_request::<lsp_request!("textDocument/definition")>(code_action_params);
 
-        if let Some(goto_definition_response) = goto_definition_response {
-            if let GotoDefinitionResponse::Scalar(location) = goto_definition_response {
+        match response {
+            Some(GotoDefinitionResponse::Scalar(location)) => {
+                report.push_str("---\n");
                 report.push_str(&peek_selection(&cairo, &location.range));
-            } else {
-                panic!("Unexpected GotoDefinitionResponse variant.")
             }
-        } else {
-            panic!("Goto definition request failed.");
+            Some(GotoDefinitionResponse::Array(locations)) => {
+                for location in locations {
+                    report.push_str("---\n");
+                    report.push_str(&peek_selection(&cairo, &location.range));
+                }
+            }
+            Some(GotoDefinitionResponse::Link(_)) => {
+                panic!("unexpected GotoDefinitionResponse::Link");
+            }
+            None => {
+                report.push_str("None");
```

Comment:
Do we expect any failures here, shouldn't we panic here? Didn't find any "None" in tests

Reply:
my next unpublished PR will add a `None` variant - for inline macro exprs (#116)

Reply:
Please refrain from doing it in the future 🙏 It makes it so:
1. Changes are not self-contained
2. Review is actually harder (I have to predict the future/look up the next PR)

Reply:
sorry, thought this will be clear to you